src/crs.jl

```diff
@@ -137,3 +137,53 @@ Base.convert(::Type{CRS}, crs::GFT.CoordinateReferenceSystemFormat) = CRS(crs)
 
 # Maybe enable later, based on https://github.com/JuliaGeo/GeoFormatTypes.jl/issues/21
 # Base.convert(T::Type{<:GFT.CoordinateReferenceSystemFormat}, crs::GFT.CoordinateReferenceSystemFormat) = T(CRS(crs))
+
+
+
+
+"""
+    identify(obj; authority = nothing, options = nothing)
+
+Returns the a list of matching reference CRS or nothing, and the confidence values (0-100) 
+or nothing. This is a high level interface to proj_identify.
+
+Inputs:
+   obj -- Object of type CRS. Must not be NULL
+   authority [nothing] -- Authority name, or NULL for all authorities (e.g. "EPSG")
+   allmatches [false] -- if true return all matches, if false return top match
+   options -- Placeholder for future options. Should be set to nothing.
+"""
+function identify(obj; authority=nothing, allmatches=false, options=nothing)
+
+    out_confidence = Ref(Ptr{Cint}(C_NULL))
+    if isnothing(authority)
+        #set authority to C_NULL
+        authority = C_NULL
+    end
+
+    if isnothing(options)
+        #set authority to C_NULL
+        options = C_NULL
+    end
+
+    pj_list = Proj.proj_identify(obj, authority, out_confidence, options)
+
+    # was a match found?
+    if pj_list == C_NULL
```

Comment:
Do you get this on error or also on no match found? If it's only the former it's better to throw an error here. If the latter, let's return an empty vector of the same type as normal.

Reply:
Proj.proj_identify returns C_NULL if not match is found. We now return an empty list